docs/start.rst

```diff
@@ -57,4 +57,8 @@ Calling the bootstrap function is generally accomplished by connecting it to the
 .. tip::
 
    If ``bootstrap: true`` is in the Thebelab configuration, this will be triggered
-   automatically upon page load.
\ No newline at end of file
+   automatically upon page load.
+
+
+.. include::
+   security.rst
```

Comment:
maybe add it to the index toc instead of included in another page?

Reply:
I was thinking the same

Reply:
Can do, I wasn't sure that we wanted it in the main TOC, so opted for here.

Will add it there now.

Reply:
![image](https://user-images.githubusercontent.com/913249/96350177-be995200-1071-11eb-9422-804c5b2c6615.png)


Thoughts?

Reply:
Maybe just call the page "security considerations" and create a sub header for this more  in case there are other security considerations to document